common/source/docs/common-gps-ardusimple.rst

```diff
@@ -0,0 +1,48 @@
+.. _common-gps-ardusimple:
+
+==========
+ArduSimple
+==========
+
+ArduSimple provide a gamme of GNSS receivers based on Ublox F9P module. Their modules allow to have RTK configuration, RTK Heading, and RTK Correction Service (NTRIP Caster).
```

Comment:
This paragraph could be changed to:

ArduSimple provides a number of UBlox F9P based GNSS receivers that support RTK and GPS-for-Yaw.  Using RTK allows ArduPilot with centimeter level accuracy which is much more accurate than a normal GPS.